controllers/network/dnsmasq_controller.go

```diff
@@ -445,6 +446,14 @@ func (r *DNSMasqReconciler) reconcileNormal(ctx context.Context, instance *netwo
 	//Update status with LoadBalancerIPs
 	instance.Status.DNSAddresses = instance.Spec.ExternalEndpoints[0].LoadBalancerIPs
 
+	//Update status with Cluster Addresses
+	poolName := instance.Spec.ExternalEndpoints[0].IPAddressPool
+	serviceName := fmt.Sprintf("%s-%s-%s", dnsmasq.ServiceName, instance.Name, poolName)
+	svc, err := service.GetServiceWithName(ctx, helper, serviceName, instance.Namespace)
+	if err == nil && svc != nil {
```

Comment:
should probably also handle `if err !=nil` ? otherwise it stops with reconcile successful.

Reply:
ack, done